README.md

```diff
@@ -40,7 +40,7 @@ You can convert your newly generated mnemonic with following hardhat task.
 Please replace the `[MENONIC]` with your mnemonic.
 
 ```shell
-npx hardhat address --mnemonic [MNEMONIC]
+npx hardhat address --mnemonic "[MNEMONIC]"
```

Comment:
@bryan-bisonai How about changing it to

```suggestion
npx hardhat address --mnemonic ${mnemonic}
```

Otherwise, it looks like a string with square brackets inside.